paasta_tools/cli/cmds/status.py

```diff
@@ -489,6 +519,11 @@ def report_status_for_cluster(
         in SSH_ONLY_INSTANCE_CONFIG
     ]
 
+    tron_jobs = [
```

Comment:
Why does this need to be done separately? Can you add Tron to HTTP_ONLY_INSTANCE_CONFIG?

Reply:
tron is not in deployed_instances/actual_deployments. It's less misleading in my opnion to add tron sperately than adding tron to deployed instances, as tron is not really deployed. Or I could rewrite the code as it's kinda dirty right now but that is too time consuming.